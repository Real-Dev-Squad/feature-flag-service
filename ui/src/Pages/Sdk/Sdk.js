import Layout from "../../Components/Layout/Layout";

export default function Sdk(){
    return(
        <Layout>
            <h1>Sdk</h1>
        </Layout>
    )
}