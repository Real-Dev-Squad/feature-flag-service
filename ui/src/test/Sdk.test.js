import { render } from '@testing-library/react';
import Sdk from '../Pages/Sdk/Sdk';

describe('<Sdk/> component', () => {
    it('component should render',() => {
        render(<Sdk />);
    });

    it('snapshpt should match',() => {
        const { container } = render(<Sdk />);
        expect(container).toMatchSnapshot();
    });
})