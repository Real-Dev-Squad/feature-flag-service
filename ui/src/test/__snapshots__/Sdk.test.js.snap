// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sdk/> component snapshpt should match 1`] = `
<div>
  <header
    class="py-2 px-8 h-20 flex items-center bg-rds-primary text-slate-100"
  >
    <img
      class="rounded-full h-12 block mx-2"
      src="https://res.cloudinary.com/realdevsquad/image/upload/v1649837643/profile/EYNHBSK7riBKY775OOMB/mfzbgspqgvj6unz34unm.jpg"
    />
    <h1
      class="text-lg"
    >
      Pallab Sonowal
    </h1>
    <button
      class="py-2 px-4 rounded-lg ml-auto bg-white text-rds-primary hover:bg-slate-400"
    >
      Logout
    </button>
  </header>
  <main
    class="flex flex-row my-4"
  >
    <nav
      class="px-3"
    >
      <ul />
    </nav>
    <section>
      <h1>
        Sdk
      </h1>
    </section>
  </main>
  <footer
    class="my-8 w-screen flex justify-center text-xs"
  >
    <span
      class="ml-2"
    >
      Copyright 2022-2023
    </span>
    <a
      class="ml-2 hover:underline"
      href="http://www.google.com"
    >
      RDS Website
    </a>
    <a
      class="ml-2 hover:underline"
      href="http://www.google.com"
    >
      Discord server
    </a>
  </footer>
</div>
`;
